pkg/accounts/service/etag_verify_generate_service.ts

```diff
@@ -0,0 +1,29 @@
+import { encodeHex } from 'std/encoding/hex';
+import { Account } from '../model/account.ts';
+
+export class EtagVerifyService {
+  /**
+   * Compare Etag with Etag in given account.
+   * @param account
+   * @param etag
+   * @returns if success: true, if failure: false
+   */
+  async compare(account: Account, etag: string): Promise<boolean> {
```

Comment:
[may]
`compare`だと文章的におかしい気がします. `verify`といった命名のほうがわかりやすいかと.
```suggestion
  async verify(account: Account, etag: string): Promise<boolean> {
```